Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoMeraki.yaml

```diff
@@ -0,0 +1,103 @@
+Parser:
+  Title: ASIM Authentication parser for Cisco Meraki
+  Version: '0.1'
+  LastUpdated: Jun 12 2023
+Product:
+  Name: Cisco Meraki
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
```

Comment:
Please add documentation link for Cisco Meraki.

Reply:
Done